christfellowshipchurch/web-app-v2
# Updated subtitle for 'My Groups'

### About
This PR redirects a user who is not in any groups to the groups page.

### Test Instructions
1. Ensure no current group enrollment 
2. Click on "click here" under "My Groups" and you should be at the groups page

### Screenshots
Before:
<img width="1246" alt="Groups:prayers issue" src="https://github.com/christfellowshipchurch/web-app-v2/assets/93110503/0818f43e-7af9-4ff6-b400-ca4f8806da3d">
After:
<img width="1365" alt="Screenshot 2023-07-06 at 3 03 51 PM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/93110503/47b1008b-e753-4063-88ae-9fc33459624f">


### Related Tickets
[CFDP-2598](https://christfellowshipchurch.atlassian.net/browse/CFDP-2598)


[CFDP-2598]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2598?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
